internal/bot/client_mock.go

```diff
@@ -72,6 +72,16 @@ func (mock *ClientMock) GetUserInfoContext(ctx context.Context, userID string) (
 	return result.(*slack.User), args.Error(1)
 }
 
+func (mock *ClientMock) GetUsersInConversationContext(ctx context.Context, params *slack.GetUsersInConversationParameters) ([]string, string, error) {
+	var (
+		args   = mock.Called(ctx, params)
+		list   = args.Get(0)
+		cursor = args.Get(1)
+		err    = args.Get(2)
```

Comment:
To access  the error you can use the function `Error()` of args:

```suggestion
		err    = args.Error(2)
```